graphql_compiler/schema_merging/utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+from graphql.language.visitor import Visitor, visit
+
+
+class SchemaError(Exception):
+    pass
```

Comment:
Nit: It would be preferable to document in a docstring line when this error could be raised. Consider the programmer who is presented with such an error. After reading the error message, and possibly the code surrounding the raising code, the next place they'd look would be the error type definition. They would greatly be aided if the docstring here told them things like what circumstances could produce this error.

If you choose to add a docstring here, the `pass` will become unnecessary and can be removed, since the docstring will provide the necessary "indented statement" that the Python parser requires.

Reply:
Also consider changing this to `something a bit more specific like `SchemaMergingError` if possible.